packages/react/src/components/header/components/headerActionBarItem/HeaderActionBarItem.module.scss

```diff
@@ -1,25 +1,25 @@
 @use "../../../../styles/common.scss";
 
 button.actionBarItem {
+  align-self: center;
   appearance: none;
   background: transparent;
   border: none;
   cursor: pointer;
   padding: 0;
 
-  @include common.focusVisible(
-    var(--header-focus-outline-color),
-    var(--header-focus-outline-width),
-    var(--header-focus-outline-border-radius),
-    calc(-1 * var(--header-focus-outline-width))
-  );
+  &:focus-visible {
+    border: none;
+    outline: var(--header-focus-outline-width) solid var(--color-coat-of-arms);
+    outline-offset: var(--header-focus-outline-width);
+  }
```

Comment:
Something has happened here to cause the focus style be amiss for logo and service name 😮 [Check in localhost this story](http://localhost:6006/?path=/story/components-header--minimal-with-localization) where logo and service name have hrefs defined and should be focusable. 
**Edit:** [Here's](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1054/commits/dcfc353bb95fd2dffd32c50f0fbbb294700fa358) the reason, specifically the focus mixin removal from class `.titleAndLogoContainer` and not providing alternative styles.

Reply:
And for whatever reason the focus order goes wonky only on mobile when the login icon is provided 😩 The focus order goes from Search to Login and then backwards to Menu, and after that to first language item.

Reply:
Reason was that login was ordered with css `order` property, but it doesn't change tab order. I made `fixedRightPosition` -prop for item, if it is used item is positioned to right side of the action bar (this is for login button)

Reply:
Something about this `fixedRightPosition` prop rubs me the wrong way a bit. It only works for one item, it doesn't work if given to multiple items. Maybe if the name is more descriptive? But then it feels a bit awkward to have this matter for just one use case. I don't know, maybe it's ok. Maybe a better name for this is `separated`? As the item literally gets separated from the rest.